indra/statements.py

```diff
@@ -1074,7 +1074,7 @@ class Statement(object):
 
     Parameters
     ----------
-    evidence : list of :py:class:`Evidence`
+    evidence : None or :py:class:`Evidence` or list of :py:class:`Evidence`
```

Comment:
What would cause evidence to be None? If a statement is synthetic it should just be an empty list.

Reply:
Never mind!